$OpenBSD: patch-src_polkitbackend_polkitbackendconfigsource_c,v 1.1 2010/07/18 15:41:22 ajacoutot Exp $

- Configuration reload on every query
779c0153fc0bd3c2e302dac1979d17638f054229
Set has_data to true after the data is loaded to prevent excessive
reloading of config files.

--- src/polkitbackend/polkitbackendconfigsource.c.orig	Wed Mar 10 18:46:19 2010
+++ src/polkitbackend/polkitbackendconfigsource.c	Sun Jul 18 17:28:52 2010
@@ -386,6 +386,7 @@ polkit_backend_config_source_ensure (PolkitBackendConf
     }
 
   source->priv->key_files = g_list_reverse (source->priv->key_files);
+  source->priv->has_data = TRUE;
 
  out:
   g_list_foreach (files, (GFunc) g_object_unref, NULL);
